.github/workflows/release.yml

```diff
@@ -410,14 +410,34 @@ jobs:
           rm -v release/._ArduinoCreateAgent*.dmg
           mv -v ArduinoCreateAgent-windows*/* release/
 
+      - name: VirusTotal Scan
+        id: virustotal_step
+        uses: crazy-max/ghaction-virustotal@v2
+        with:
+          vt_api_key: ${{ secrets.VIRUSTOTAL_API_KEY }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we trust this 3rd party action?

Reply:
The only thing it does is uploading binaries to virustotal. I don't see the problem

Reply:
Probably the github token is not required

Reply:
Is it used to update the release artifacts?

Reply:
Yeah, but it's not used because the action is not triggered by a release